tweaks.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# This is free and unencumbered software released into the public domain.
```

Comment:
I doubt the compatibility of this license with Proton's

Reply:
I copied it from `filelock.py` distributed with Proton. But I probably should change hashbang to require `python3`.

Reply:
Changed to python3 (I am not going to write code that works in both python2 and python3 at the same time, it's a hassle).